requirements/env_unidist.yml

```diff
@@ -0,0 +1,54 @@
+name: modin_on_unidist
+channels:
+  - conda-forge
+dependencies:
+  - pandas==1.5.1
+  - numpy>=1.18.5
+  - pyarrow>=4.0.1
+  - fsspec
+  - xarray
+  - Jinja2
+  - scipy
+  - pip
+  - s3fs>=2021.8
+  - feather-format
+  - lxml
+  - openpyxl
+  - xlrd
+  - matplotlib
+  - sqlalchemy>=1.4.0
+  - pandas-gbq
+  - pytables
+  - msgpack-python
+  - psutil
+  - pytest>=6.0.1
+  - pytest-benchmark
+  - pytest-cov>=2.10.1
+  - pytest-xdist>=2.1.0
+  - packaging
+  - coverage
+  - pygithub
+  - rpyc==4.1.5
+  - cloudpickle
+  - boto3
+  - scikit-learn
+  - pymssql
+  - psycopg2
+  - mypy
+  - pandas-stubs
+  - fastparquet
+  # for release script
+  - pygit2
+  - pip:
+      - unidist[mpi]>=0.2.1
```

Comment:
conda hasn't picked up yet the release.

Reply:
What is the estimated date of?

Reply:
has it picked it up now?

Reply:
moved to installation with conda, should be working now.